zk/src/archive/web/js/zk/dom.js

```diff
@@ -800,8 +800,8 @@ zjq.prototype = {
 		if ((w = this.$()) && w._scrollbar) {// support a fake scrollbar
 			return w._scrollbar.hasVScroll();
 		}
-		var n, v;
-		return (n = this.jq[0]) && (v = n.clientWidth) && (v = n.offsetWidth - v) > 11 ? v : 0;
+		var n, v;//zk-3938: if zoom-in, the v will be smaller than 11.
+		return (n = this.jq[0]) && (v = n.clientWidth) && (v = n.offsetWidth - v) > 1 ? v : 0;
```

Comment:
1. Why 11 or 1? Can't it be 0?
2. `hasHScroll` has a similar one, both should be modified together.

Reply:
ok